project/BenchmarkWorkFlow.scala

```diff
@@ -0,0 +1,43 @@
+import sbtghactions.GenerativePlugin.autoImport.{UseRef, WorkflowJob, WorkflowStep}
+
+object BenchmarkWorkFlow {
+  def apply(): Seq[WorkflowJob] = Seq(
+    WorkflowJob(
+      runsOnExtraLabels = List("zio-http"),
+      id = "runBenchMarks",
+      name = "Benchmarks",
+      oses = List("centos"),
+      cond = Some("${{ github.event_name == 'push'}}"),
+      env = Map("GITHUB_TOKEN" -> "${{secrets.ACTIONS_PAT}}"),
+      steps = List(
+        WorkflowStep.Checkout,
+        WorkflowStep.Use(
+          UseRef.Public("actions", "checkout", s"v2"),
+          Map(
+            "repository" -> "amitksingh1490/FrameworkBenchmarks",
```

Comment:
```suggestion
            "repository" -> "dream11/FrameworkBenchmarks",
```
Either this ... or can we do something else?